src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -131,10 +142,11 @@ private static String launchAndJoinCommand(Launcher launcher, List<String> args)
     ByteArrayOutputStream cmdLogStream = new ByteArrayOutputStream();
     int status = launcher.launch().cmds(args).stderr(cmdLogStream).stdout(cmdLogStream).join();
     if (status != 0) {
-      LOGGER.log(
-          Level.SEVERE, String.format("kubectl command log: %s", cmdLogStream.toString(CHARSET)));
+      String logs = cmdLogStream.toString(CHARSET);
+      LOGGER.log(Level.SEVERE, String.format("kubectl command log: %s", logs));
       throw new IOException(
-          String.format("Failed to launch command args: %s, status: %s", args, status));
+          String.format(
+              "Failed to launch command args: %s, status: %s. Logs: %s", args, status, logs));
```

Comment:
This should be printed out in the "Console Output" for the build. The logs were added so that the error from running the command would also be shown on the command line for `mvn hpi:run`.

I think it's good to see the list of arguments explicitly in addition to seeing the full command, but we might also want to see the command here so it's visible on the command line.